libkernelflinger/tpm2_security.c

```diff
@@ -279,7 +286,7 @@ EFI_STATUS tpm2_write_nvindex(TPMI_RH_NV_INDEX nv_index,
 
 	ret = Tpm2FlushContext(session_handle);
 	if (EFI_ERROR(ret)) {
-		efi_perror(ret, L"tpm2_write_nvindex - FlushContext failed");
+		efi_perror(ret, L"TPM2 write NV index - FlushContext failed");
```

Comment:
Everywhere in the code where you want to print the function name in the error message use the `"%a", __func__` instead.  If the function name changes or if this code move to another function, you don't have to update it.  Also, it makes git blame more efficient finding where it comes from in the first place.